manifests/0000_20_kube-apiserver-operator_06_deployment.yaml

```diff
@@ -89,6 +91,9 @@ spec:
                   apiVersion: v1
                   fieldPath: metadata.namespace
                 path: namespace
+      - name: service-ca-bundle
```

Comment:
This has to be optional

Reply:
Actually, maybe it does not have to be but I'd rather see the operator be capable to start w/o it

Reply:
we can mark it as optional, other precedences don't, but that's fine in this case. If missing, metrics controller will record `PrometheusClientFailed` and will eventually succeed during subsequent syncs.

Reply:
fixed